src/builtin/numbers.scrbl

```diff
@@ -376,15 +391,15 @@ either integers or decimals, with an optional exponent.
 ~-6.022e-23
 }
 
-Roughnums cannot be made arbitrarily precise. The absolute value
-ranges between 0 and
-1.7976931348623157e+308 (JS’s Number.MAX_VALUE) with a
-granularity of 5e-324 (JS’s Number.MIN_VALUE).
+@pyret{Roughnum}s cannot be made arbitrarily precise. The absolute value
+ranges between 0 and 1.7976931348623157e+308 (JS’s Number.MAX_VALUE) with a
```

Comment:
Let's go with JavaScript, please. That's the language's name, so it avoids confusion down the road if "JS" comes to mean something else.